tests/TourGuide.tex

```diff
@@ -0,0 +1,85 @@
+\section{Tour guide [SSPL only]}
+The robot guides spectators to the audience area and answer their questions after explaining what's @Home about.
+
+\subsection{Focus}
+This test focuses in safe outdoor navigation, people detection, gesture recognition, unconstrained natural language processing, and Human-Robot Interaction
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location:} This test takes place outside the arena in a public space close to the @Home area.
+
+	\item \textbf{Other people:} There are no restrictions on other people walking by or standing around throughout the complete task.
+
+	\item \textbf{In Parallel:} This test can run in parallel, with several teams tested simultaneously.
+\end{itemize}
+
+\subsection{Task}
+
+\begin{enumerate}
+	\item \textbf{Start:} The robot starts at a designated starting position, and waits for referee to give the start signal. When the referees start the time, the team is allowed to (briefly) provide some remarks about the robot's operation. After the instruction, the referee gives the start signal.\\
```

Comment:
Indeed, I should say:

> The robot waits at a designated starting position for the referee to give the start signal...

This is meant to prevent delays in explanations. The test time is running so the longer (and less intuitive) the instructions are given to the referee, the less time the robot has to accomplish the task. Our ideal is that robot has no issue if a human is following it.